tools/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What is this tsConfig file used for?

Reply:
For the files in `tools` the enviroment should be `node` but not `browser`.

There will be some wrong message in my `webstorm` without this `tsconfig`.